cogs/base_cog.py

```diff
@@ -5,17 +5,45 @@
 from custom_errors  import InvalidTypeError
 
 class BaseCog(commands.Cog):
-    def __init__(self, bot, category):
+    def __init__(self, bot):
         self.bot = bot
-        self.category = category
-
         self.data_path = get_json_path("data")
 
-    # Get data from JSON file
     async def get_data(self):
+        """
+            Return the data stored inside the ``data.json`` file.
+
+            Returns
+            -------
+            json_data: :class:`dict`
+                the JSON object.
+        """
+
```

Comment:
[This is how you comment on multiple lines](https://github.blog/changelog/2019-10-01-multi-line-references-and-comments-beta/)

Reply:
diff-57d1e283c8336272e5cd7ef43601cf82da069f0c4961e0e46bddbf3c1b4c6ad3R13

Reply:
https://github.com/Hao-bot-5000/project-nort/pull/8/commits/2c3e275e9cd5b29f6f035cf68d41c55166e3c122

Reply:
https://github.com/Hao-bot-5000/project-nort/pull/8/files#diff-57d1e283c8336272e5cd7ef43601cf82da069f0c4961e0e46bddbf3c1b4c6ad3R13-R15

Reply:
https://github.com/Hao-bot-5000/project-nort/pull/8/files#diff-57d1e283c8336272e5cd7ef43601cf82da069f0c4961e0e46bddbf3c1b4c6ad3R14